src/tests/integration/omneo/countries/list-countries.test.ts

```diff
@@ -0,0 +1,52 @@
+import { describe, expect, test, afterAll } from 'vitest'
+import { Omneo } from '../../../../omneo'
+import simpleOmneoRequest from '../../../lib/simple-omneo-request'
+import { CountryInput, CountryResponse } from '../../../../types'
+import { getName, getIso2, getIso3, getIsoNumeric } from './util'
+
+const omneo = new Omneo({
+  tenant: process.env.OMNEO_TENANT as string,
+  token: process.env.OMNEO_TOKEN as string
+})
+const CREATED_COUNTRIES_HANDLES : number[] = []
+
+describe('Countries list', () => {
+  test('SDK Get Countries', async () => {
+    const payload: CountryInput = {
+      name: getName(),
+      iso_2: getIso2(),
+      iso_3: getIso3(),
+      iso_numeric: getIsoNumeric(),
+      sort_order: null
+    }
+    const response = await simpleOmneoRequest('POST', '/countries', payload)
+    CREATED_COUNTRIES_HANDLES.push(response.data.id)
+
+    const countriesRes: CountryResponse = await omneo.countries.list({
+      'filter[name]': payload.name
+    })
+    const { data: countries } = countriesRes
+    const filterCountries = countries.filter(d => d.name === payload.name)
+    expect(filterCountries.length).toBeGreaterThan(0)
```

Comment:
this should actually be equal to 1 right?
Since you're filtering to your specific random name that you've just created
There should never be more than 1, if there is there is an issue

I would check that the length is === 1 and that the first member of the array `id` = your response simpleOmneoRequest id

Reply:
yeah, great feedback, I will update this.